omim2obo/main.py

```diff
@@ -365,13 +378,20 @@ def omim2obo(use_cache: bool = False):
                 'gene_id': gene_mim, 'phenotype_label': p_lab, 'mapping_key': p_map_key, 'mapping_label': p_map_lab})
 
     # - Add relations (subclass restrictions)
-    exclusions_p_mim_orcid_map = get_d2g_exclusions_by_curator()
+    exclusions_p_mim_orcid_map: Dict[str, Optional[URIRef]] = get_d2g_exclusions_by_curator()
+    protections_p_mim_orcid_map: Dict[str, Optional[URIRef]] = get_d2g_protections_by_curator()
```

Comment:
`protections_p_mim_orcid_map` --> `protected_p_mim_orcid_map`

Reply:
Ok, pushing a commit now but likewise with this let me know if you want exclusions renamed to excluded. I think It is called exclusions in our other repos so the "ions" instead of "ed" is technically more consistent, but that is a `nit`.